sale_order_line_date/__manifest__.py

```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
```

Comment:
@OpenCode why removing everywhere the ```coding``` line?

Reply:
https://github.com/OCA/account-invoicing/pull/341#discussion_r160036766

Reply:
ok